### ISSUE TITLE
Convenient ? operator {{ hero?.address?.street }} instead of v-if="hero && hero.address"

### ISSUE THREAD

**Issue by smith64fx** [vuejs/vue#4675]
Hello Vue Lovers! :D

Today I realized that there is something I really miss in VueJS that I liked in Angular2.

It's the ? operator. It's explained very good here:
https://angular.io/docs/ts/latest/guide/template-syntax.html#!#safe-navigation-operator

I give a quick example.
Everybody knows this problem.

If I get a hero async and I want to display a subproperty like this:
```
The null hero's name is {{ hero.address.street }}
```
.. and the hero is not yet loaded, I will get an error like this:

```
TypeError: Cannot read property 'address' of undefined.
or
TypeError: Cannot read property 'address' of street
```

Usually we have to make a 
```
v-if="hero && hero.address"
```
or something like that.

That's why in Angular2 they have the so-called "? operator"
So you can do something like this:

```
The null hero's name is {{ hero?.address?.street }}
```

So it will do the if statements for us in a really convenient way.

Is it possible to integrate something like this in VueJS? 
This would be amazing! :)

**Comment by simplesmiler** [comment]
Closing as duplicate of https://github.com/vuejs/vue/issues/4638
TLDR: we don't want to introduce non-standard syntax to expressions.
